src/components/TableHeadCell.js

```diff
@@ -89,7 +89,8 @@ class TableHeadCell extends React.Component {
 
   render() {
     const { isSortTooltipOpen, isHintTooltipOpen } = this.state;
-    const { children, classes, options, sortDirection, sort, hint, print } = this.props;
+    const { children, classes, options, sortDirection, sort, hint, print, cellHeaderProps } = this.props;
+    const { className, ...otherProps } = cellHeaderProps;
```

Comment:
Similar to the other comment about checking for the `className` property, as the dev will get an unhelpful error if they return the wrong type.

Reply:
I'm not sure I follow. Check that it's set to a string?